modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -279,13 +279,13 @@ init_statement: |2
 
 === `schema_evolution`
 
-Options to control schema updates as new columns are added to the pipeline.
+Options to control schema updates as new tables and columns are added to the pipeline.
```

Comment:
```suggestion
Options to control schema updates as new tables and columns when schemas are added to the pipeline.
```

Reply:
I'm not sure this is necessary because you have to add a schema to the pipeline (i.e. it's a required value).

Reply:
Agreed we should not add `when schemas`. There is always a schema.

Reply:
What does 'are added to the pipeline' refer to?

Reply:
I inferred the meaning from the next option description:

> Whether schema evolution is enabled. Set to `true` to allow the automatic creation of new tables and columns in the <<schema,specified schema>> when they are added to your pipeline


Note 'when they are added to your pipeline'.

Reply:
Updated. Hopefully, the initial description of the `schema_evolution` object and what happens when `schema_evolution` is enabled now fit together.